src/main/io/cms.c

```diff
@@ -76,9 +77,9 @@
 #include "io/cms_ledstrip.h"
 
 // Forwards
-void cmsx_InfoInit(void);
-void cmsx_FeatureRead(void);
-void cmsx_FeatureWriteback(void);
+long cmsx_InfoInit(void);
```

Comment:
I was thinking of returning a pointer, say a pointer to a string, from these in-menu functions. Then my collapsing memory told me L=P for all 32 and 64-bit architectures under C99.